Modules/Core/Transform/test/itkResampleInPlaceImageFilterTest.cxx

```diff
@@ -145,5 +148,7 @@ itkResampleInPlaceImageFilterTest(int argc, char * argv[])
   TransformType * rawPointerTransform = transform.GetPointer();
   rawPointerTransform->ApplyToImageMetadata(inputImage);
 
+  ITK_EXERCISE_BASIC_OBJECT_METHODS(filter, ResampleInPlaceImageFilter, ImageToImageFilter);
```

Comment:
Although given the variables in this filter this is not likely, I think that it is desirable to put this right after the instantiation of the filter to be able to spot problems that can otherwise go unnoticed (e.g. uninitialized member variables).